help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
+|--------------|---------|---|------|-----|------|--------|------|
+| bool         | *       | * | .    | .   | .    | *      | +    |
+| date         | *       | * | *    | *   | .    | *      | +    |
+| datetime     | *       | . | .    | *   | .    | *      | +    |
+| int          | *       | * | *    | *   | *    | *      | +    |
+| string       | *       | * | *    | *   | *    | *      | +    |
+| uuid         | *       | * | *    | .   | .    | *      | .    |
+| text         | *       | . | .    | .   | *    | *(8)   | .    |
+| --------(9)  |         |   |      |     |      |        |      |
+| dict(1)      | *       | * | *    | .   | .    | *      | *    |
+| object(2)    | *       | . | .    | .   | .    | *      | .    |
+| dict_list(3) | *       | * | *(7) | .   | .    | *      | .    |
+| list(4)      | *       | . | .    | .   | .    | *      | .    |
+
+- `null` - operator poprawny tylko dla typów nullable
+- `=` - porównanie z wartością tego samego typu, int-int, string-string, itp., brak możliwości porównania z null ani z
```

Comment:
Dopisz tutaj o tym że case-insensitive i że ignoruje polskość znaków. Wtedy też będzie wynikało że te same reguły aplikują się do cmp i in. I jeszcze do like, ale to byś powinien osobno napisać przy like.